### ISSUE TITLE
Regresión Lineal

### ISSUE THREAD

**Issue by kbp2** [kbp2/cienciadedatos#5]
import pandas as pd
from sklearn.linear_model import LinearRegression
import matplotlib.pyplot as plt

# Crear DataFrame
df = pd.DataFrame({
    "Edad": [22, 25, 28, 30, 35, 40],
    "Ingresos": [3000, 3200, 3500, 3700, 4000, 4200]
})

# Separar variables
X = df[["Edad"]]   # predictor
y = df["Ingresos"] # variable objetivo

# Crear y ajustar el modelo
modelo = LinearRegression()
modelo.fit(X, y)

# Mostrar coeficientes
print("Pendiente (m):", modelo.coef_[0])
print("Intercepto (b):", modelo.intercept_)

# Graficar datos y predicción
plt.scatter(df["Edad"], df["Ingresos"], color="blue", label="Datos reales")
plt.plot(df["Edad"], modelo.predict(X), color="red", label="Modelo lineal")
plt.title("Regresión lineal: Edad vs Ingresos")
plt.xlabel("Edad")
plt.ylabel("Ingresos")
plt.legend()
plt.grid(True)
plt.show()
